README.md

```diff
@@ -30,13 +32,30 @@ Click on "Use this Template"
 
 ![Use this Template](https://docs.github.com/assets/images/help/repository/use-this-template-button.png)
 
-Name the repository, and provide a description. We recommend using the following naming conventions:
+Name the repository, and provide a description.
+
+Depending on the plugin relationship with the OpenSearch organization we currently recommend the following naming conventions and optional follow-up checks:
+
+### Official plugins
```

Comment:
I don't love the name "Official". How about "Plugins Released with OpenSearch", of which there are 2 kinds, native and non-native?

Reply:
I was following naming pattern from InstallPluginCommand.java, see the [JavaDoc](https://github.com/opensearch-project/OpenSearch/blob/b251d2b565b918708a1612ec16d1916122c7805d/distribution/tools/plugin-cli/src/main/java/org/opensearch/plugins/InstallPluginCommand.java#L112) and [var name](https://github.com/opensearch-project/OpenSearch/blob/b251d2b565b918708a1612ec16d1916122c7805d/distribution/tools/plugin-cli/src/main/java/org/opensearch/plugins/InstallPluginCommand.java#L167) (also in [online docs](https://opensearch.org/docs/latest/opensearch/install/plugins/)). Shall we rename in the install tool source code as well?

I would use "Native" and "Thirdparty" plugins. WDYT?

Reply:
I see. That can be a separate PR. Let's leave it this way here, consider following up?

Reply:
> consider following up?

Yes, naming should be addressed in different PR.